libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/Microsoft.Bot.Builder.Adapters.Twilio.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Placeholder task: we need a Microsoft.Bot.Builder.Adapters.Twilio.Tests XUnit project under the tests/Adapters folder in the solution.

Reply:
ACK. This is our next step.